plugins/adminchat.sma

```diff
@@ -216,20 +216,18 @@ public cmdSayAdmin(id)
 	log_amx("Chat: ^"%s<%d><%s><>^" chat ^"%s^"", name, userid, authid, message[1])
 	log_message("^"%s<%d><%s><>^" triggered ^"amx_chat^" (text ^"%s^")", name, userid, authid, message[1])
 	
-	if (is_user_admin(id)) // no diff here if admins have g_AdminChatFlag access or not, but we don't want to print "PLAYER"
-		format(message, charsmax(message), "(%L) %s :  %s", id, "ADMIN", name, message[1])
-	else
-		format(message, charsmax(message), "(%L) %s :  %s", id, "PLAYER", name, message[1])
-
 	get_players(players, inum, "ch")
 	
-	for (new i = 0; i < inum; ++i)
+	for (new bool:is_sender_admin = !!is_user_admin(id), i = 0; i < inum; ++i)
```

Comment:
I prefer `!= 0` instead of `!!`.

Reply:
Is there a difference?

Reply:
It's just more friendly. It's easier to read "is not equal to" than "not not" :P.  You get it.

Reply:
@OciXCrom yeah, but added/changed code should be as good as possible.